src/main/java/com/example/controller/BookController.java

```diff
@@ -0,0 +1,40 @@
+package com.example.controller;
+
+import com.example.dto.BookDto;
+import com.example.dto.CreateBookRequestDto;
+import com.example.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/books")

```

Comment:
extract common to all endpoints /api prefix to application.properties (server servlet context-path)